assets/package.json

```diff
@@ -17,6 +17,7 @@
   "dependencies": {
     "@rehooks/component-size": "^1.0.3",
     "@sentry/react": "^5.19.2",
+    "@sentry/webpack-plugin": "^1.12.0",
```

Comment:
If this package is just used during webpack building, and doesn't need to be shipped to the client, it should go in `devDependencies`.